Multitouch Support/VoodooI2CMultitouchInterface.cpp

```diff
@@ -54,6 +54,18 @@ bool VoodooI2CMultitouchInterface::handleIsOpen(const IOService *forClient ) con
     return false;
 }
 
+bool VoodooI2CMultitouchInterface::setProperty(const OSSymbol* key, OSObject* object) {
+    bool result = super::setProperty(key, object);
+    if (result) {
```

Comment:
I usually like to return in case of an error, to avoid excessive indentation, but it's just coding style and we don't have such convention :)

Reply:
Thanks for the advice.